db/migrate/20200422190916_remove_prohibited_advertiser_ids_from_properties.rb

```diff
@@ -0,0 +1,10 @@
+class RemoveProhibitedAdvertiserIdsFromProperties < ActiveRecord::Migration[6.0]
+  def up
+    remove_column :properties, :prohibited_advertiser_ids
+  end
+
+  def down
+    add_column :properties, :prohibited_advertiser_ids, :bigint, default: [], null: false, array: true
+    add_index :properties, :prohibited_advertiser_ids, using: :gin
+  end
+end
```

Comment:
This will need to be a phased deploy where we only drop this column after the data migration has been run successfully.

Reply:
Ignore as this is addressed by #1209